### DISCUSSION TITLE
脚注のリンクが正しく機能しない

### DISCUSSION THREAD

**Discussion by munierujp** [increments/qiita-discussions#371] [要望・アイデア・不具合]
脚注本体のid属性が空になっていて、本文中にある脚注へのリンクをクリックしても飛べないようになっています。

<img width="1259" alt="スクリーンショット 2022-12-08 8 23 49" src="https://user-images.githubusercontent.com/20086673/206318740-908ca2c5-7a0d-4a2a-9a95-4bc7baa1dc21.png">

https://qiita.com/Qiita/items/c686397e4a0f4f11683d#%E8%84%9A%E6%B3%A8

**Comment by tanaka-asami415** [comment]
@munierujp 
ご報告ありがとうございます！ご不便をおかけして申し訳ありません🙇
こちら修正対応中ですので、少しだけお待ちいただけると幸いです！

**Reply by kyntk** [reply]
@munierujp お待たせしました！
脚注の修正が完了しました！ 🙏
